src/qt/window.cpp

```diff
@@ -518,7 +518,10 @@ bool wxWindowQt::SetCursor( const wxCursor &cursor )
     if (!wxWindowBase::SetCursor(cursor))
         return false;
 
-    GetHandle()->setCursor(cursor.GetHandle());
+    if ( cursor.IsOk() )
+        GetHandle()->setCursor(cursor.GetHandle());
+    else
+        GetHandle()->setCursor(wxCursor(wxCURSOR_DEFAULT).GetHandle());
```

Comment:
Looking at [Qt docs](http://doc.qt.io/qt-5/qwidget.html#cursor-prop), should we be using `unsetCursor()` here instead? Because as written this would override the default window cursor (e.g. I-shaped one for a `wxTextCtrl`) with the default cursor which is not the intention.

Also, if we do need to use `setCursor`, I think it would be better/simpler/more efficient to just use default-constructed `QCursor`. According to the documentation, this is the default cursor, so we could do just
```suggestion
        GetHandle()->setCursor(QCursor());
```
couldn't we?

Reply:
Yep - we will take a look at this and resubmit.